src/pages/resources/faq.md

```diff
@@ -85,8 +85,8 @@ For a complete list of supported platforms, please read the [latest SDK versions
 
 ### What OS and platform versions are supported?
 
-* Android versions 4.4 or later (API levels 19 or later)
-* iOS versions 10 or later
+* Android versions 5 or later (API levels 21 or later)
```

Comment:
Should we add supported OS versions in this page itself https://developer.adobe.com/client-sdks/home/current-sdk-versions/

React Native and Flutter already mention supported versions in their section.

It will be better to add either one consolidated table showing all supported platforms  (ios, android, roku, react, flutter ..) and supported versions. And also adding new platform and/or updating any will be easier. We can link the sdk versions page with table in here. What do you think?

Reply:
Can you create a GH issue to add this to current versions page?

Reply:
@praveek https://github.com/AdobeDocs/aep-mobile-sdkdocs/issues/744